src/services/User.service.js

```diff
@@ -0,0 +1,21 @@
+import db from '../models';
+
+/**
+ * Class User services creates a middleware
+ */
+class UserServices {
```

Comment:
I think we should avoid classes and static methods and defer to objects:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Details_of_the_Object_Model